### PR TITLE
LPS-182194 | Master

### PR DESCRIPTION
**Pull request by WYZ-Issac**
https://issues.liferay.com/browse/LPS-182194

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileEntry.testcase [hunk 1]
```diff
@@ -3064,6 +3064,96 @@ definition {
...
+			mimeType = "application/msword",
+			sourceFileName = "Document_2.doc");
+
+		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
+
+		ManagementBar.setFilterAndOrder(filterBy = "Tags...");
+
+		DMDocument.selectTags(tagNameList = "tag1,tag2");
+
+		var key_dmDocumentTitle = ${dmDocumentTitle};
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMFileEntry.testcase:3092]: Is this needed? It's declared again two times below

File: portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro [hunk 2]
```diff
@@ -893,6 +893,24 @@ definition {
 		}
 	}
 
+	macro AssertSelectedTags {
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/macros/DMDocument.macro:896]: Please be more specific with the assertion you are making (is element present, not visible, etc)

**Review by austinchiang — CHANGES_REQUESTED** [review]
Please review and resend any changes

**Comment by WYZ-Issac** [comment]
ci:test:sf

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by WYZ-Issac** [comment]
ci:test:sf

**Comment by WYZ-Issac** [comment]
ci:test:relevant

**Comment by austinchiang** [comment]
LGTM

**Comment by austinchiang** [comment]
ci:forward

**Pull request closed without merging**